samples/ziti-spring-boot-client/src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+
+logging:
+  level:
+    org.openziti: TRACE
```

Comment:
this is an overkill, it'll be overwhelming for anyone new trying it
I'd set it to ERROR/WARNING, and add instructions for changing it if troubleshooting is required